src/mixxx.cpp

```diff
@@ -261,6 +261,7 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
                              m_pRecordingManager);
     m_pPlayerManager->bindToLibrary(m_pLibrary);
 
+    m_pLibrary->scan();
```

Comment:
OK, sorry, I was late to see the comment of @daschuer and I made PR in quickness, I need to know now where is the missing list of the files which are missing from where i can find it